packages/expo-modules-core/android/src/compose/expo/modules/kotlin/views/ExpoComposeView.kt

```diff
@@ -38,7 +38,7 @@ abstract class ExpoComposeView<T : ComposeProps>(
   var testID: String? = null
 
   @Composable
-  abstract fun Content(modifier: Modifier = Modifier)
+  abstract fun Content(modifier: Modifier)
```

Comment:
I am experiencing crashes when rendering compose components due to this default; removing it seems to resolve the issue without causing any new problems.

I can send up a separate PR for this fix if necessary.

Reply:
same here – let's remove it then!